### ISSUE TITLE
Order of code execution

### ISSUE THREAD

**Issue by JeetRoy97** [yumoxu/lqsum#2]
Kindly provide the order in which I should execute the code to replicate the results.

**Comment by ateffal** [comment]
Yes the order please,  and also see that there is no **data/cnndm/** folder under **lqsum** as is described in BPE instructions : 

`DATASET=train && IO=source && RAW=~/lqsum/data/cnndm/${DATASET}.${IO} && TGT=~/lqsum/data/cnndm_bpe/origin/${DATASET}.bpe.${IO} && cd ~/lqsum/guided_summarization/bart && . z_bpe.sh ${RAW} ${TGT}`
